integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/mortgage/MortgageSpark.scala

```diff
@@ -422,6 +422,10 @@ object AggregatesWithJoin {
 
 object Main {
   def main(args: Array[String]): Unit = {
+    if (args.length < 4 || args.length > 5) {
+        System.err.println("Usage:<sparkversion> <perfpath> <acqpath> <outputpath> [csv|orc|parquet]")
+        System.exit(1)
+    }
```

Comment:
nit: we probably don't need an explicit exit and we can handle argument validation with the require idiom
```suggestion
    require ((4 until 5) contains args.length, "Usage:<sparkversion> <perfpath> <acqpath> <outputpath> [csv|orc|parquet]")
```

Reply:
`require` will throw with a stacktrace, that's not what we want here.

Reply:
if anything it adds diagnostics, a matter of taste imo. not a big deal to me

Reply:
Fixed 'case _ =>' Nothing value, @gerashegalov @jlowe, thanks a lot for you help